README.md

```
@@ -1,2 +1,72 @@
+
+
+## Limitations
+
+This is a simple demonstration server intended for educational purposes. It doesn't support:
+
+- Multiple concurrent connections
+- HTTP methods other than GET
+- Dynamic content generation
+- Security features
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)